x-pack/plugins/actions/server/raw_action_schema.test.ts

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
can we rename these files `raw_connector_schema*` :-). And basically change every text reference of `action` to `connector` in the files.

Reply:
done